core/mintController.js

```diff
@@ -1,50 +1,57 @@
 import { makePrivateName } from '../util/PrivateName';
 
 export function makeBasicMintController() {
-  // Map from purse or payment to the rights it currently
-  // holds. Rights can move via payments
-
-  // purse/payment to amount
-  let rights = makePrivateName();
-
-  function recordPayment(src, payment, amount, srcNewRightsAmount) {
-    rights.set(src, srcNewRightsAmount);
-    rights.init(payment, amount);
+  // An asset can either be a purse or payment. An asset controller
+  // keeps track of either all of the purses (purseController) or all
+  // of the payments (paymentController).
+  function makeAssetController(type) {
```

Comment:
Don't call this `type`. At some point this code will grow to have types in some programming sense of the word type, so we should avoid it for other uses.